### PR TITLE
Changes on the Screen to display prices should Exclude VAT

### PR DESCRIPTION
**Pull request by DeepikaDK001**

### CODE REVIEW

File: tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Models/PriceCalculationDetailsModelTests.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        public void Constructor_WhenTieredAndSingleFixed_ReturnsCorrectTitle(
+            CataloguePriceType priceType,
+            CataloguePriceCalculationType calculationType)
+        {
+            var model = new PriceCalculationDetailsModel(
+                CatalogueItemType.Solution,
+                priceType,
+                calculationType);
+
+            if (priceType == CataloguePriceType.Tiered && calculationType != CataloguePriceCalculationType.SingleFixed)
```
Comment (ghost) [inline tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Models/PriceCalculationDetailsModelTests.cs]: Let's split this out to two tests

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/Index.cshtml [hunk 2]
```diff
@@ -288,11 +288,11 @@
             @($"£{Model.Order.TotalOneOffCost():N2}")
         </pdf-summary-list-row>
 
-        <pdf-summary-list-row label-text="Total monthly cost">
+                            <pdf-summary-list-row label-text="Total monthly cost">
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/Index.cshtml]: Indentation to be removed

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/Index.cshtml [hunk 3]
```diff
@@ -288,11 +288,11 @@
...
             @($"£{Model.Order.TotalMonthlyCost():N2}")
         </pdf-summary-list-row>
 
-        <pdf-summary-list-row label-text="Total annual cost">
+                            <pdf-summary-list-row label-text="Total annual cost">
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/Index.cshtml]: Indentation to be removed

File: tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Models/PriceCalculationDetailsModelTests.cs [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        public void Constructor_WhenNotCumulative_ReturnsCorrectHeading(
+            CataloguePriceType priceType,
+            CataloguePriceCalculationType calculationType)
+        {
+            var model = new PriceCalculationDetailsModel(
+                CatalogueItemType.Solution,
+                priceType,
+                calculationType);
+
+            var expectedHeading = $"The total price for this Catalogue Solution will be calculated as a {priceType.ToString().ToLowerInvariant()}";
```
Comment (ghost) [inline tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Models/PriceCalculationDetailsModelTests.cs:96]: Separate tests

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Orders/Views/FundingSource/FundingSources.cshtml [hunk 5]
```diff
@@ -98,10 +99,11 @@
 				</nhs-table-row-container>
 				}           
 				<nhs-table-row-container>
-					<nhs-table-cell><b>Total</b></nhs-table-cell>
+                    <nhs-table-cell><b>Total</b></nhs-table-cell>
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Orders/Views/FundingSource/FundingSources.cshtml:102]: Indentation and formatting

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Orders/Views/FundingSource/FundingSources.cshtml [hunk 6]
```diff
@@ -98,10 +99,11 @@
...
 					<nhs-table-cell>@($"£{Model.OrderItemsLocalOnly.Sum(oi => CalculateTotalCost(oi)).ToString("N2")}")</nhs-table-cell>
 				</nhs-table-row-container>
 			</nhs-table>
+            <p><b>All prices exclude VAT.</b></p>
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Orders/Views/FundingSource/FundingSources.cshtml]: Indentation and formatting

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Orders/Views/FundingSource/FundingSources.cshtml [hunk 7]
```diff
@@ -77,6 +77,7 @@
 					<nhs-table-cell></nhs-table-cell>
 				</nhs-table-row-container>
 			</nhs-table>
+            <p><b>All prices exclude VAT.</b></p>
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Orders/Views/FundingSource/FundingSources.cshtml]: Indentation and formatting

**Review by bjss-colin — CHANGES_REQUESTED** [review]

**Review by imranbashir01 — APPROVED** [review]

**Review by bjss-colin — APPROVED** [review]

**Pull request merged**